quickstart/docker-compose.yml

```diff
@@ -0,0 +1,322 @@
+version: "3"
+services:
+  apollo-router:
+    image: ghcr.io/apollographql/router:v1.25.0
+    ports:
+    - 4004:4004
+    environment:
+    - APOLLO_ROUTER_SUPERGRAPH_PATH=/repo/dev/apollo-federation/supergraph.graphql
+    - APOLLO_ROUTER_CONFIG_PATH=/repo/dev/apollo-federation/router.yaml
+    volumes:
+    - ${HOST_PROJECT_PATH:-.}/${GALOY_QUICKSTART_PATH:-vendor/galoy-quickstart}/dev:/repo/dev
+  oathkeeper:
+    image: oryd/oathkeeper:v0.40.4-distroless
+    ports:
+    - 4002:4455
```

Comment:
Is it possible to split out the port mapping into a separate docker-compose file similar to how we do it for our docker-compose file for this repo. We need to be able to disable the port mapping so that when the bats tests runs on concourse they only happen within a docker network and dont map to any ports on the concourse worker.

Reply:
We could but let me ask - have you observed this being an issue? I think since we restrict our docker host to only run 1 job at a time (via concourse locks) this should no longer be an issue.

Reply:
The hiding-of-port-mapping stuff we did was to be able to run multiple jobs that need docker at once (so as not to have port collisions). But with only 1 job running at any time (which is the case now) I think we don't have that problem anymore.

Reply:
I didn't know that we switched to one job at a time. I don't think we need it then.